### PR TITLE
Remove triple backticks from ops.model docs since Sphinx doesn't like them

### LINKED ISSUE #599: Docs for is_ready does not format correctly
https://github.com/canonical/operator/blob/c441f1a105bd6fca91d2558d0cfcb92f6c21f7e8/ops/model.py#L1076-L1086

Sphinx does not seem to interpret "```" as a code block.

### PR DESCRIPTION
**Pull request by rbarry82**
As the header, really.

Closes #599

### CODE REVIEW

File: ops/model.py [hunk 1]
```diff
@@ -1074,7 +1074,6 @@ def is_ready(self) -> '_ContainerReady':
          raising exceptions.
 
         Example:
```
Comment (benhoyt) [inline ops/model.py:1076]: This still doesn't render as a monospaced code block in ReST. Use the `::` syntax, so `Example::` followed by a blank line:

```suggestion
        Example::
        
```

Comment (benhoyt) [reply ops/model.py:1076]: There's another instance of this (and triple back ticks) below.

In general, the docs need a bit of a syntax go-over, but oh well, another day...

**Review by benhoyt — CHANGES_REQUESTED** [review]

File: ops/model.py (hunk 1 above)
Comment (benhoyt) [reply ops/model.py:1076]: BTW, you can build the docs with `./build_docs` (assuming you're in a venv with `requirements-dev.txt` libraries installed).

**Comment by rbarry82** [comment]
Fixed/updated. That make target is super useful. Thanks @benhoyt !

File: ops/model.py [hunk 2]
```diff
@@ -1073,8 +1073,7 @@ def is_ready(self) -> '_ContainerReady':
          from the underlying Pebble operations will log error messages rather than
          raising exceptions.
 
-        Example:
-            ```
+        Example::
```
Comment (benhoyt) [inline ops/model.py:1076]: I believe you need a blank line after the `Example::` line for it to render as a monospaced code block.

**Review by benhoyt — APPROVED** [review]

**Review by dstathis — APPROVED** [review]
:+1:

**Pull request merged**